packages/playwright-core/src/server/bidi/bidiBrowser.ts

```diff
@@ -373,6 +374,19 @@ export class BidiBrowserContext extends BrowserContext {
   }
 
   async doUpdateRequestInterception(): Promise<void> {
+    if (this.requestInterceptors.length > 0 && !this._interceptIdPromise) {
+      this._interceptIdPromise = this._browser._browserSession.send('network.addIntercept', {
```

Comment:
Let's await for the command result and save the intercept id, instead of a promise. The method is `async` so Playwright contract guarantees that it will be awaited for before the context is used again. In the worst case, we'll end up sending several commands